src/util/getUrl.ts

```diff
@@ -11,7 +11,7 @@ export const getGitHubConfigurationUrl = (urlParams: UrlParams): string => {
 };
 
 export const getJiraMarketplaceUrl = (jiraHost: string): string =>
-	`https://${jiraHost}/plugins/servlet/upm/marketplace/plugins/com.github.integration.production`;
+	`${jiraHost}/plugins/servlet/ac/com.atlassian.jira.emcee/discover#!/discover/app/com.github.integration.productio`;
```

Comment:
missing last character in url

Reply:
Ugh. Thought I fixed this last night. :sigh: